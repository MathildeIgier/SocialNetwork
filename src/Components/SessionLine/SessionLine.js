import React from 'react';
import { View, Text } from 'react-native';
import Icon from 'react-native-vector-icons/Ionicons';

import styles from './SessionLineStyles';

export default function SessionLine({ value, icon }) {
  return (
    <View style={styles.sessionLineContainer}>
      <View style={styles.iconContainer}>
        <Icon name={icon} size={24} color="#fff" />
      </View>
      <Text style={styles.itemValue}>{value}</Text>
    </View>
  );
}
