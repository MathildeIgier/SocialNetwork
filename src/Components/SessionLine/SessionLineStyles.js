import { StyleSheet } from 'react-native';
import { Colors, ApplicationStyles } from '../../Themes';

export default StyleSheet.create({
  ...ApplicationStyles.screen,
  sessionLineContainer: {
    flexDirection: 'row',
    paddingTop: 15
  },
  iconContainer: {
    width: 20,
    alignItems: 'center'
  },
  itemValue: {
    color: Colors.snow,
    fontSize: 16,
    paddingTop: 3,
    marginLeft: 10
  }
});
