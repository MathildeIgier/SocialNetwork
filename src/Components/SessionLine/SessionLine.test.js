// __tests__/Intro-test.js
import React from 'react';
import renderer from 'react-test-renderer';

import SessionLine from './SessionLine';

test('renders correctly', () => {
  const tree = renderer.create(<SessionLine />).toJSON();
  expect(tree).toMatchSnapshot();
});
