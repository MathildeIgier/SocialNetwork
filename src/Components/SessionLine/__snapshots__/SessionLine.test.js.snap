// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingTop": 15,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "width": 20,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 24,
          },
          undefined,
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "marginLeft": 10,
        "paddingTop": 3,
      }
    }
  />
</View>
`;
